assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -130,19 +131,12 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi }) => {
           query,
         });
 
-        const response = await dataAdapter.get(path, {
-          parse: false,
-          cache: 'no-cache',
-        });
-
-        const totalPages =
-          response.headers && parseInt(response.headers.get('X-WP-TotalPages'));
+        const response = await dataAdapter.get(path);
 
-        const totalStoriesByStatus =
-          response.headers &&
-          JSON.parse(response.headers.get('X-WP-TotalByStatus'));
+        const totalPages = response.totals && parseInt(response.totals.all);
```

Comment:
Could you add the ability to get the total number of pages as part of the response here, totals.all would work in a pinch - we'll just divide by the total per page on the front end if need be but I'd like to keep that logic in the API if possible. 

So for instance if I have a total of 30 stories and I'm getting 24 per request then I have a total page count of 2. We're using this number to decide if we keep calling for more stories so it's important to have.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/2933/commits/d4f42cbb9da6a8edf730fa6bc03ad45c98bff5c1